std/node/_fs/_fs_appendFile.ts

```diff
@@ -39,13 +39,13 @@ export function appendFile(
           //TODO rework once https://github.com/denoland/deno/issues/4017 completes
           notImplemented("Deno does not yet support setting mode on create");
         }
-        const file = await Deno.open(pathOrRid, getOpenOptions(flag));
+        const file = Deno.openSync(pathOrRid, getOpenOptions(flag));
         rid = file.rid;
       }
 
       const buffer: Uint8Array = new TextEncoder().encode(data);
 
-      await Deno.write(rid, buffer);
+      Deno.writeSync(rid, buffer);
```

Comment:
The function is now sync... The goal is to apply this lint rule without changing the behavior.

Reply:
right , all sync methods removed 👍🏻